### PR TITLE
Jspann 4 org detail page

### PR DESCRIPTION
**Pull request by RubenOlano**
First draft of what the organization page looks like

### CODE REVIEW

**Comment by Jake3231** [comment]
I am seeing some compilation issues related to merging:
```
./components/DirectoryOrgs.tsx
Error: 
  x Merge conflict marker encountered.
    ,----
 11 | <<<<<<< HEAD
    : ^^^^^^^
    `----

  x Expression expected
    ,----
 12 | <div className="md:w-full h-auto flex flex-col justify-between items-center p-5 m-5 rounded-2xl">
    :                                                                                                 ^
    `----

  x Merge conflict marker encountered.
    ,----
 13 | =======
    : ^^^^^^^
    `----

  x Expression expected
    ,----
 14 | <div className="w-full h-auto flex flex-col justify-between items-center p-5 m-5 rounded-2xl bg-white shadow-md hover:shadow-lg hover:cursor-pointer">
    :                                                                                                                                                      ^
    `----

  x Merge conflict marker encountered.
    ,----
 15 | >>>>>>> cc11936 (Update grid to be more responsive and further break down org cards and org tags with small updates)
    : ^^^^^^^
    `----

  x Expression expected
    ,----
 16 | <h4 className="text-lg font-bold text-center">{org.name}</h4>
    :                                              ^
    `----

  x Expected a semicolon
    ,----
 16 | <h4 className="text-lg font-bold text-center">{org.name}</h4>
    :                                                   ^
    `----

  x Expected ',', got '.'
    ,----
 16 | <h4 className="text-lg font-bold text-center">{org.name}</h4>
    :                                                   ^
    `----

Caused by:
    0: failed to process input file
    1: Syntax Error
    ```

**Comment by Jake3231** [comment]
It may be easier to start a new branch off of `develop`, then copy the organization page into that new branch and re-wire it to the directory if merge conflicts prove to be a pain.

**Comment by Jake3231** [comment]
Looks like it now compiles, though it has a different iteration of the directory UI. Is that an expected change? Also, is it expected that the org page is accessible only by navigating to `localhost:3000/organization`?

**Comment by Jake3231** [comment]
That definitely makes sense. I was mostly referring to that clicking the cards seemingly does not open the page. Would it also be an option to pass the org ID as a URL parameter (`{url}/organization?id=12345`)?

**Comment by RubenOlano** [comment]
We can pass it through query parameters if that is easier. I can implement the rerouting in a separate pull request

File: .github/ISSUE_TEMPLATE/bug_report_issue_template.yml [hunk 1]
```diff
@@ -50,8 +47,8 @@ body:
   - type: textarea
     id: logs
     attributes:
-        Please copy and paste any relevant log output. This will be
-        automatically formatted into code, so no need for backticks.
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```
Comment (Jake3231) [inline .github/ISSUE_TEMPLATE/bug_report_issue_template.yml:51]: Is it expected that the bug report template is changed with this PR?

**Comment by Jake3231** [comment]
The Org page looks really good, though it looks like many other (older) changes are included with this PR. We'll want to resolve that to reduce the scope of this PR to be sure it's only changing the organization page and related components. I'm looking to see if there's a simple way to do that.

File: .github/ISSUE_TEMPLATE/bug_report_issue_template.yml (hunk 1 above)
Comment (RubenOlano) [reply .github/ISSUE_TEMPLATE/bug_report_issue_template.yml:51]: I think I'll just look into manually changing that back

**Comment by RubenOlano** [comment]
> The Org page looks really good, though it looks like many other (older) changes are included with this PR. We'll want to resolve that to reduce the scope of this PR to be sure it's only changing the organization page and related components. I'm looking to see if there's a simple way to do that.

Yeah, let me know if there's something I can do to help with that. Else I can just start from a fresh branch

**Comment by Jake3231** [comment]
@RubenOlano I may have resolved it. Please let me know if it looks like the expected changes are still shown as expected.

**Comment by RubenOlano** [comment]
image
image
New looks

**Review by Jake3231 — APPROVED** [review]
The org detail page design looks really good!

File: pages/organization.tsx [hunk 2]
```diff
@@ -1,5 +1,47 @@
-import React from "react";
+import DirectoryOrgHeader from '../components/DirectoryOrgHeader';
```
Comment (Jake3231) [inline pages/organization.tsx:1]: I just noticed this - is it expected that the DirectoryOrgHeader name is used for this component in the org page?

Comment (RubenOlano) [reply pages/organization.tsx]: Just renamed that to just "OrgHeader"

**Pull request merged**